src/components/Refero/ReferoSidebar.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { getSidebarElements, generateSectionContent } from '../../locales/referoSidebarResources';
+import { Questionnaire } from '../../types/fhir';
+
+type Props = {
+    questionnaire: Questionnaire | string;
+};
+
+const ReferoSidebar = ({ questionnaire }: Props) => {
+    const [isSidebarViewEnabled, setIsSidebarViewEnabled] = React.useState(false);
+    const sidebarData = getSidebarElements(questionnaire ? questionnaire : '');
+
+    return (
+        <div
+            style={{
+                position: 'fixed',
+                top: '70px',
+                bottom: 0,
+                right: 0,
+                padding: 8,
+                backgroundColor: '#fff',
+                zIndex: 90,
+                transition: 'transform .5s ease,-webkit-transform .5s ease',
+                width: '27.5rem',
+                borderLeft: '2px #AAA solid',
+                transform: isSidebarViewEnabled ? 'translateX(0)' : 'translateX(100%)',
+            }}
```

Comment:
Couldn't this be placed in a scss file as well?

Reply:
I guess it could, the reason i left it inline is because transform depends on the "isSidebarViewEnabled" state.